clamd/clamd.c

```diff
@@ -568,6 +568,12 @@ int main(int argc, char **argv)
                 ret = 1;
                 break;
             }
+
+            if (stat(CL_ENGINE_TMPDIR, &sb) != 0 && !S_ISDIR(sb.st_mode)) {
+                logg(LOGG_ERROR, "%s is not a valid dir\n", CL_ENGINE_TMPDIR);
```

Comment:
It would be useful to explain why we care about this directory, in case it isn't apparent. 
Something like:
`"The temporary directory is configured to be '%s' but this directory does not exist or is not a valid directory.\n"`

Reply:
updated the log to `Current configuration of TemporaryDirectory: %s does not exist, or is not valid`